app/controllers/hq/orphans_controller.rb

```diff
@@ -23,6 +24,17 @@ def update
 
 private
 
+  def load_scope
+    @orphans_all = Orphan.all().size
```

Comment:
I've read [this](http://web.archive.org/web/20100210204319/http://blog.hasmanythrough.com/2008/2/27/count-length-size) article, and stay that if I use `size` I don't mind to care about the choose, because it changes between the state in that moment. I'd like to take the more efficient option tho.